proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -52,6 +53,30 @@ service Msg {
 
   // AllowAskDenom is a governance operation which authorizes a new ask denom to be used in sell orders
   rpc AllowAskDenom (MsgAllowAskDenom) returns (MsgAllowAskDenomResponse);
+
+  // CreateBasket creates a bank denom which wraps credits.
```

Comment:
```suggestion
  // CreateBasket creates a bank denom which wraps credits.
  // Baskets are managed and created by Regen Ledger governance (REGEN holders).
  // This means that basket creation and basket composition updates goes through
  // governance processes and module parameters.
```

adding 578bbbdc24a28cd8f0cfb11ea9c3f137c2a5358c

Reply:
But how is governance controlling this? Don't we need to modify the API.